### ISSUE TITLE
Fix and Support for Offline ADT Pulse Gateway

### ISSUE THREAD

**Issue by mrjackyliang** [mrjackyliang/homebridge-adt-pulse#15]
This is pulled from one of the anonymous logs at [console.re](https://console.re/nsJJTsUAB6AUwcCNc4rzuPwcW9JAuCeXG8DeGtVypRpT2gFw). The object explains a `doorWindow` accessory with incorrect tags and state.
```json
{
    "id": "sensor-70",
    "name": "Kitchen Door",
    "tags": "sensor",
    "index": "20VER1",
    "state": "devStatUnknown"
}
```

Still determining why this is happening to some users. The correct `tags` are `sensor,doorWindow`, while the `state` should normally be `devStatOK`, `devStatOpen`, or `devStatTamper`.

### Notes
- Appears on entire `getZoneStatus` request.
- Happens when the ADT Pulse Gateway is offline.

**Comment by mrjackyliang** [comment]
Recreated the scenario by disconnecting the ADT Pulse Gateway. The same incorrect tags and state will show up.

**Comment by mrjackyliang** [comment]
This issue is now fixed in `v1.6.3`. Closing for now.
